### ISSUE TITLE
issue with config file for redis 2.8.4

### ISSUE THREAD

**Issue by mreinsch** [redis-store/testing#1]
When running tests, I see the following:

```
** Invoke redis:replication:start (first_time)
** Execute redis:replication:start

*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 16
>>> 'dbfilename tmp/dump.rdb'
dbfilename can't be a path, just a filename

*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 16
>>> 'dbfilename tmp/node-one-dump.rdb'
dbfilename can't be a path, just a filename

*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 16
>>> 'dbfilename tmp/node-two-dump.rdb'
dbfilename can't be a path, just a filename
```

**Comment by mreinsch** [comment]
changing https://github.com/redis-store/testing/blob/master/lib/redis-store/testing/config/redis.conf#L16 and the other configs to not include paths for the dbfilename does indeed solve the problem and makes the tests pass

**Comment by radar** [comment]
This has now been fixed.

**Comment by bhuvan-rk** [comment]
Hi @radar 

Can you elaborate on this fix ? I just tried an upgrade from 2.6.16 to 3.0.7 and I encountered the same error. 

In which version has this fix gone in ?

https://stackoverflow.com/questions/36752546/dbfilename-cant-be-a-path-just-a-filename

**Comment by tubbo** [comment]
@bhuvan-rk This gem is not user-facing, and the only reason why we have it is to set up an actual Redis instance for testing the various redis-store gems. It's basically just common functionality as a development dependency, and thus is not actually included into your project. The StackOverflow post indicates that you are not using the Redis configuration provided [in the testing configs](https://github.com/redis-store/testing/commit/31148dbeb7ac75fb22dd17085b26e921eac12542).
